microcosm_flask/conventions/discovery.py

```diff
@@ -73,7 +73,7 @@ def discover():
 
 
 @defaults(
-    name="",
+    name="hal",
```

Comment:
I had to use a non-empty name here because Flask was very unhappy resoling the endpoint `.discovery.v1`. I kept the root path for compatibility.